ui/titlepage/templates/TitlePage.jinja

```diff
@@ -1,92 +1,69 @@
 {%- extends "invenio_theme/frontpage.html" %} {% block head_links %} {% endblock
-  head_links %} {% block body %}
-  
-  <main class="pusher">
-    <Navbar />
-    <div class="main-page__half--background">
-    <div class="horiz-div main-page__main-container">
-      <div class="vert-div main-page__main-container__content">
-        <!-- page title -->
-  
-        <div class="horiz-div main-page__title">
-          <div>
+head_links %} {% block body %}
+
+<main class="pusher main-page">
+  <div class="ui segment hero vertical left aligned">
+    <div class="ui container">
+      <BrandLogo />
```

Comment:
<img width="716" alt="Screenshot 2024-03-07 at 16 27 37" src="https://github.com/VSCHT/restaurovani/assets/646743/ab579de2-bf3e-4e61-a27c-86d4145dc52d">

Reply:
I don't have this issue

Reply:
Yet it is an issue :)

<img width="197" alt="Screenshot 2024-03-08 at 14 05 53" src="https://github.com/VSCHT/restaurovani/assets/646743/4c61640f-36db-4a94-8754-b5a03bdeb82b">

Reply:
U mne lokalne to ma camelcase name